op-monitorism/cmd/monitorism/cli.go

```diff
@@ -202,13 +228,13 @@ func DrippieMain(ctx *cli.Context, closeApp context.CancelCauseFunc) (cliapp.Lif
 	log := oplog.NewLogger(oplog.AppOut(ctx), oplog.ReadCLIConfig(ctx))
 	cfg, err := drippie.ReadCLIFlags(ctx)
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse drippie config from flags: %w", err)
+		return nil, fmt.Errorf("Failed to parse drippie config from flags: %w", err)
```

Comment:
This change may affect parser that are currently relying on a specific string format for generating alerts. If you plan to change this maybe is good to add teams using those messages for review before changing them.

Reply:
Oh for now the parser is not used for the alert, I will make sure if the message are not case sensitive there.